### PR TITLE
add LA plugins

### PR DESCRIPTION
**Pull request by megastef**

### CODE REVIEW

File: docs/logagent/output-plugin-http.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
+title: Ship logs via http or https
+description: Sends log messages as http or https post request
+
+## Output Plugin: HTTP
+
+Post logs to web services via `http` or `https`.
```
Comment (adnanrahic) [inline docs/logagent/output-plugin-http.md]: POST logs to web services via `HTTP` or `HTTPS`.

File: docs/logagent/output-plugin-http.md [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    format: ldjson
+    # maximum number of events to buffer
+    # 1 - each event creates a separate http request
+    # >1 - multiple events in each http request
+    maxBufferSize: 1
+    # flush interval in seconds
+    flushInterval: 1
+    # add tags before logs are shipped
+    tags:
+      token: SPM_TOKEN
```
Comment (adnanrahic) [inline docs/logagent/output-plugin-http.md]: Does this work with MONITORING_TOKEN as well?

File: docs/logagent/plugins.md [hunk 3]
```diff
@@ -1,4 +1,4 @@
-title: Logagent plugins 
+!title: Logagent plugins
```
Comment (adnanrahic) [inline docs/logagent/plugins.md]: what is this `!` ?

File: docs/logagent/plugins.md [hunk 4]
```diff
@@ -48,9 +50,14 @@ Logagent features a modular architecture. Each input or output module is impleme
 | [output-files](output-plugin-files)     | output                    | Stores parsed messages files. Log rotation and dynamic file name generation are supported.                                                                  |
 | [output-clickhouse](output-plugin-clickhouse)       | output                   | Sends parsed messages to Yandex ClickHouse DB |
 | [logagent-output-kafka](output-plugin-kafka)       | output                   | Sends parsed messages to Apache Kafka topics. 3rd party module. 3rd party module.                                                             |
+| [output-http](output-plugin-clickhouse)       | output                   | Sends parsed messages via HTTP or HTTPS |
 | [slack-webhook](output-plugin-slack)      | output                    | Sends parsed messages to Slack chat. Should be combined with SQL filter plugin or filter function to define alert criterias. |
 | [@sematext/logagent-nodejs-monitor](https://www.npmjs.com/package/@sematext/logagent-nodejs-monitor) | other | Monitors server and  nodejs metrics of the Logagent process using [spm-agent-nodejs](https://www.npmjs.com/package/spm-agent-nodejs) |
 
+## Find plugins on NPM
```
Comment (adnanrahic) [inline docs/logagent/plugins.md]: NPM => npm

File: docs/logagent/plugins.md [hunk 5]
```diff
@@ -48,9 +50,14 @@ Logagent features a modular architecture. Each input or output module is impleme
...
 | [output-clickhouse](output-plugin-clickhouse)       | output                   | Sends parsed messages to Yandex ClickHouse DB |
 | [logagent-output-kafka](output-plugin-kafka)       | output                   | Sends parsed messages to Apache Kafka topics. 3rd party module. 3rd party module.                                                             |
+| [output-http](output-plugin-clickhouse)       | output                   | Sends parsed messages via HTTP or HTTPS |
 | [slack-webhook](output-plugin-slack)      | output                    | Sends parsed messages to Slack chat. Should be combined with SQL filter plugin or filter function to define alert criterias. |
 | [@sematext/logagent-nodejs-monitor](https://www.npmjs.com/package/@sematext/logagent-nodejs-monitor) | other | Monitors server and  nodejs metrics of the Logagent process using [spm-agent-nodejs](https://www.npmjs.com/package/spm-agent-nodejs) |
 
+## Find plugins on NPM 
+
+Developers of 3rd party plugins publish logagent plugins in the NPM registry.
```
Comment (adnanrahic) [inline docs/logagent/plugins.md]: Developers of 3rd party plugins publish logagent plugins in the NPM registry. 
=>
Developers of 3rd party plugins publish Logagent plugins in the npm registry.

File: docs/logagent/output-plugin-http.md (hunk 2 above)
Comment (megastef) [reply docs/logagent/output-plugin-http.md]: Tags are just text fields, we can remove `toke`n from the example. So people are not confused with any of Sematext App tokens.

File: docs/logagent/plugins.md (hunk 3 above)
Comment (megastef) [reply docs/logagent/plugins.md]: A typo. See https://squidfunk.github.io/mkdocs-material/extensions/metadata/

File: docs/logagent/input-kubernetes-events.md [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+- A working `kubectl` config
+- Or run Loagent in a pod, role bindings to access k8s API pods, events, and namespaces 
+
+### Configuration
+
+```yaml
+
+# Receive Kubernetes events via HTTP server
+input:
+   kubernetesEvents:
```
[the remark this answers is not in the document]
Comment (megastef) [reply docs/logagent/input-kubernetes-events.md]: 2 spaces according to YAML spec.

**Pull request merged**